ui/packages/components/src/PostgresIntegrations/types.ts

```diff
@@ -43,3 +43,19 @@ export type IntegrationPageContent = {
   logo: React.ReactNode;
   url: string;
 };
+
+export function parseConnectionString(connectionString: string) {
+  const regex = /postgresq?l?:\/\/(\w+):([^@]+)@([^/]+)/;
+  const match = connectionString.match(regex);
+
+  if (match) {
+    const [, username, password, host] = match;
+    return {
+      name: `Neon-${host}`,
```

Comment:
When we support other providers in the future, should this not be hard coded?

Reply:
Yep, there are a few of those today, just for the sake of speed!